### ISSUE TITLE
Can't refresh password store when using latest version of gopass

### ISSUE THREAD

**Issue by mcanevet** [camptocamp/terraform-provider-pass#12]
Config file changed in recent version of gopass. `path` is now in this format:
```
path: gpgcli-gitcli-fs+file:///home/<foo>/.password-store
```
The provider fails refreshing the state with:
```
Error: Error refreshing state: 1 error(s) occurred:                                                                                                                                              
                                                                                                                                                                                                 
* provider.pass: error refreshing password store: failed to run git pull on sub store root: failed to run command /usr/bin/git [git pull]: chdir /home/<foo>/tmp/tf/gpgcli-gitcli-fs+file:/~/
.password-store: no such file or directory
```
I tried to update dependencies and recompile to see if it fixes this bug, but compilation fails...

@raphink any idea?

**Comment by tagirb** [comment]
Seems to be fixed with 0377278d0832f4318931a29d23595c4239155d55. Right?

**Comment by mcanevet** [comment]
Yes, I close this. Thanks
